clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -2655,6 +2655,15 @@ CXXBaseSpecifier *Sema::CheckBaseSpecifier(CXXRecordDecl *Class,
       return nullptr;
     }
 
+    if (BaseType.hasQualifiers() && !isa<SubstTemplateTypeParmType>(BaseType)) {
+      auto Quals =
```

Comment:
NIT: could you please spell out the type here?
It is understandable that result is some kind of a string, however it is unclear whether it is a StringRef or a std::string, for example.

Reply:
thanks for the feedback. I've updated it to use an explicit type.